RealmJS.podspec

```diff
@@ -0,0 +1,97 @@
+require 'json'
+
+package = JSON.parse(File.read(File.expand_path('package.json', __dir__)))
+
+# Building the podspec from a local .git repository when developing - GitHub otherwise
+git_path_local = File.expand_path('.git', __dir__)
+git_path = File.directory?(git_path_local) ? "file://#{git_path_local}" : 'https://github.com/realm/realm-js.git'
+
+app_path = File.expand_path('../..', __dir__)
+node_modules_path = File.expand_path('node_modules', app_path)
+react_native_module_path = File.expand_path('react-native', node_modules_path)
+if not File.directory?(react_native_module_path)
+  raise "Couldn't find React Native node module (#{react_native_module_path}) - did you install `npm install react-native`?"
+end
+
+Pod::Spec.new do |s|
+  s.name                   = "RealmJS"
```

Comment:
Why not `Realm`?

Reply:
This is primarily because this name ends up being the product name in the xcode project, which is referenced here https://github.com/realm/realm-js/blob/master/react-native/ios/RealmReact/RealmAnalytics.mm#L69. Choosing this name I didn't have to publish an update to RealmAnalytics.mm on GitHub.

Reply:
Realm is also already taken by the obj-c Pod.